crates/router/src/routes/recon.rs

```diff
@@ -64,203 +50,9 @@ pub async fn get_recon_token(state: web::Data<AppState>, req: HttpRequest) -> Ht
         state,
         &req,
         (),
-        |state, user: ReconUser, _, _| generate_recon_token(state, user),
-        &auth::ReconJWT,
+        |state, user: UserFromToken, _, _| recon::generate_recon_token(state, user),
+        &auth::DashboardNoPermissionAuth,
```

Comment:
This is not backwards compatible right?

Reply:
It's the same as ReconJWT auth, we have removed the redundant data types